emva1288/process/results.py

```diff
@@ -788,24 +788,62 @@ def s_2_y_dark(self):
 
     @property
     def s_2_y_cav(self):
+        """Mid value of Spatial variance from image,
```

Comment:
`cav` and `rav` are very confusing, what would be a better naming?

Also `s_2_y_cav` and `s_2_y_rav` share the same description

Reply:
It's a really good question, I didn't find the good name in the emva4.0 documentation (page 15). They have the same description since they are very similar. Maybe I can add some row and column differences.

Reply:
I think it's the best